source/api/ut_runner.pks

```diff
@@ -74,10 +74,10 @@ create or replace package ut_runner authid current_user is
   /**
   * Removes cached information about annotations for objects of specified type and specified owner
   *
-  * @param a_object_owner owner of objects to purge annotations for
-  * @param a_object_type  optional type of objects to purge annotations for (defaults to 'PACKAGE')
+  * @param a_object_owner  optional - owner of objects to purge annotations for
+  * @param a_object_type   optional - type of objects to purge annotations for (defaults to 'PACKAGE')
```

Comment:
It doesn't default to PACKAGE anymore, does it?

Reply:
It is fixed now. Once you approve I'll merge it. Thanks.